### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by Gracetemitope**
In this branch, I implemented the Calculator UI

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 import './App.css';
 
-function App() {
-  return (
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  /* eslint-disable */
```
Comment (arslanbisharat) [inline src/App.js]: - [ ] Please ignore disabling eslint

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Please add .vscode to gitignore so you have a clean repo.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by arslanbisharat** [comment]
Hi there, 
I got your message on slack. Please address your issue here so we can have a history of our discussion for future code reviewer.

**Comment by Gracetemitope** [comment]
Hi!
If I ignore disabling eslint, then it'll keep throwing errors. I wouldn't mind if you could work me through the error
Screenshot (9)

**Comment by arslanbisharat** [comment]
> Hi!
> If I ignore disabling eslint, then it'll keep throwing errors. I wouldn't mind if you could work me through the error
> Screenshot (9)

Hi there, 
Thank you for asking. This is a basic requirement to not disable eslint. You need to work on this error.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @Gracetemitope, 

### Status: Approved ✔️

:+1: You have done implementing all the changes required by the previous reviewer.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- You have removed almost all the `disabling eslint`, remaining with the only one that's why I am approving this request, but make sure to use best practices to avoid those.
-  Also, I would recommend to give a better PR title so that we can understand what this request is all about.


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**